pkg/cedar/intake/translation/system_intake.go

```diff
@@ -67,7 +67,7 @@ func (si *TranslatableSystemIntake) CreateIntakeModel() (*wire.IntakeInput, erro
 		CostIncreaseAmount:          si.CostIncreaseAmount.Ptr(),
 		Contractor:                  si.Contractor.Ptr(),
 		ContractVehicle:             si.ContractVehicle.Ptr(),
-		ContractNumber:              si.ContractNumber.Ptr(),
+		ContractNumber:              helpers.PointerTo(""), // TODO: populate with linked contract numbers?
```

Comment:
@ClayBenson94 Is this code for sending an intake to Cedar?

Reply:
Yep - I forgot to migrate my comments over here from the closed PR. My fault.

> if we want to populate this, we'll need access to the dataloaders here. I tried some different ways of getting the dataloaders in this area, but i was not successful in that.
> 
> the alternative is to have a version of the getter that not a loader and use that

Reply:
^ignore please - sometimes fresh eyes helps 🙂. I pushed a change to send them the linked contract numbers

Reply:
Ah -- yeah, I suppose this is probably something we'll have to update on the CEDAR side as well.

If it's not too much work to fetch those contract numbers and join them with `,`, that might be preferred. If that's a decent LOE to get that in this part of the codebase, let me know and I can work with the CEDAR team to see if it's time to just make some larger schema changes to support how we represent contract numbers now.

Reply:
Thanks! Updated in [a720782](https://github.com/CMSgov/easi-app/pull/2460/commits/a72078261d97ac4db720fa5b43dd54fd08741bdd)